gcp/extract/lib/bundles.py

```diff
@@ -48,15 +48,12 @@ def read_region(target_regions, *args, **kwargs):
     regions : array-like
     data : array-like
     """
-    # Because someone's always tryin to ice-skate up hill
-    if isinstance(target_regions, str):
-        target_regions = [target_regions]
-
     years, regions, data = read(*args, **kwargs)
-    regions_msk = np.isin(regions, target_regions)
+    regions_msk = np.ones(regions.shape, dtype='bool')
 
-    if target_regions == ['all'] or target_regions == []:
-        regions_msk[:] = True
+    if not configs.is_allregions(config):
+        target_regions = configs.get_regions(config, regions)
+        regions_msk[:] = np.isin(regions, target_regions)
```

Comment:
Why aren't you just saying here:
```
if not configs.is_allregions(config):
    target_regions = configs.get_regions(config, regions)
    regions_msk = np.isin(regions, target_regions)
else:
    regions_msk = np.ones(regions.shape, dtype='bool')
```
? It seems weird to set up the variable just to replace it.

Reply:
Good suggestion. I can also flip the if/else and drop the `not` to make it more clear. Just pushed a commit with this change. Thanks.